frontends/web/src/components/devices/bitbox02/mnemonicpassphrase.tsx

```diff
@@ -73,8 +73,8 @@ class MnemonicPassphraseButton  extends Component<Props, State> {
                                             !mnemonicPassphraseEnabled && message && (
                                                 <p>
                                                     {
-                                                        message.split('\n').map(line => (
-                                                            <span>
+                                                        message.split('\n').map((line, i) => (
+                                                            <span key={`line-${i}`}>
```

Comment:
i think the key can be anything, also just `key={i}`, or not?

Reply:
Yes we can if there is nothing else and we are sure the order never changes, https://reactjs.org/docs/lists-and-keys.html#keys

> We don’t recommend using indexes for keys if the order of items may change.

Reply:
Using `key={i}` when the message is always the same, squash pushed here https://github.com/digitalbitbox/bitbox-wallet-app/compare/adefd672dff147b228e4bd0a6676e5a1a8186f14..0ffdf60a2d7dd90bf569daa432db50f6e03fb797

keeping a combination of line + i for Spinner

Reply:
tbh i didn't see the `line` argument. if you wanted to have it in the key, then `line-${i}` didn't do that, you want `${line}-${i}`, or maybe `[i, line]`?

Reply:
Oh, yes ofc I wanted `${line}-${i}` :P

Reply:
Thanks for catching this.